Looks like the windows stage needs more fixes;

```
go build -o _output/containerd.exe \
	-tags '' \
	-ldflags '-X github.com/containerd/cri/vendor/github.com/containerd/containerd/version.Version=4ed8d8f-TEST' \
	-gcflags '' \
	github.com/containerd/cri/cmd/containerd
can't load package: package github.com/containerd/cri/cmd/containerd: cannot find package "github.com/containerd/cri/cmd/containerd" in any of:
	C:\hostedtoolcache\windows\go\1.13.15\x64\src\github.com\containerd\cri\cmd\containerd (from $GOROOT)
	C:\Users\runneradmin\go\src\github.com\containerd\cri\cmd\containerd (from $GOPATH)
Error: mingw32-make: *** [Makefile:85: _output/containerd.exe] Error 1
Error: Process completed with exit code 1.
```

--- a/.github/workflows/ci.yml
+++ b/.github/workflows/ci.yml
@@ -19,8 +19,8 @@ jobs:
       - name: Set env
         shell: bash
         run: |
-          echo "::set-env name=GOPATH::${{ github.workspace }}"
-          echo "::add-path::${{ github.workspace }}/bin"
+          echo "GOPATH=${{ github.workspace }}" >> $GITHUB_ENV
+          echo "${{ github.workspace }}/bin" >> $GITHUB_PATH
 
       - name: Checkout cri repo
         uses: actions/checkout@v2
@@ -87,8 +87,8 @@ jobs:
       - name: Set env
         shell: bash
         run: |
-          echo "::set-env name=GOPATH::${{ github.workspace }}"
-          echo "::add-path::${{ github.workspace }}/bin"
+          echo "GOPATH=${{ github.workspace }}" >> $GITHUB_ENV
+          echo "${{ github.workspace }}/bin" >> $GITHUB_PATH
 
       - name: Checkout cri repo
         uses: actions/checkout@v2
@@ -158,16 +158,20 @@ jobs:
           path: ${{github.workspace}}\\src\\github.com\\containerd\\cri
 
       - name: Clone containerd repo
+        shell: bash
         run: |
-          bash.exe -c "GO111MODULE=off go get github.com/containerd/containerd"
+          GO111MODULE=off go get github.com/containerd/containerd
 
       - name: Configure Windows environment variables
+        shell: bash
         run: |
-          echo "::set-env name=GOPATH::$env:GITHUB_WORKSPACE"
+          echo "GOPATH=${{ github.workspace }}" >> $GITHUB_ENV
+          echo "${{ github.workspace }}/bin" >> $GITHUB_PATH
 
       - name: Build
+        shell: bash
         run: |
-          bash.exe -c "pwd && ./test/windows/test.sh"
+          pwd && ./test/windows/test.sh
         working-directory: ${{github.workspace}}\\src\\github.com\\containerd\\cri
 
       - name: Upload containerd log file

